waspc/data/Generator/templates/sdk/wasp/client/index.ts

```diff
@@ -11,4 +11,4 @@ export enum HttpMethod {
 export type Route = { method: HttpMethod; path: string }
 
 // PUBLIC API
-export { default as config } from './config'
+export { config, ClientConfig } from './config'
```

Comment:
I've read in "Effective typescript" (or something like that):

> When you export a value X of type T, you should also expose T. In other words, if users can and see a type, it's already public, so make it easily accessible.

I've followed this principle for `emailSender` as well